src/plugins/twitter/twitter.js

```diff
@@ -40,39 +40,78 @@ var componentName = "wb-twitter",
 				return;
 			}
 
-			// If the plugin container's first child element is a Twitter link...
-			if ( twitterLink && twitterLink.matches( "a.twitter-timeline" ) ) {
-				const loadingDiv = document.createElement( "div" );
-				let observer;
+			// Process each Twitter link
+			twitterLinks.forEach( function( twitterLink ) {
+				
+				// If the link doesn't specify a widget language (i.e. lacks a data-lang attribute)...
+				if ( !twitterLink.dataset.lang ) {
 
-				// Add a loading icon below the link
-				loadingDiv.className = "twitter-timeline-loading";
-				twitterLink.after( loadingDiv );
+					// Get the link's "in-page" language (i.e. it or its closest parent's lang attribute)
+					let twitterLinkLang = twitterLink.closest( '[lang]:not([lang=""])' ).getAttribute( "lang" );
 
-				// Remove the loading icon after the timeline widget appears
-				// Note: Twitter's widget script removes "a.twitter-timeline" upon filling-in the timeline's content... at which point the loading icon is no longer useful
-				observer = new MutationObserver( function( mutations ) {
+					// If the language is truthy...
+					if ( twitterLinkLang ) {
 
-					// Check for DOM mutations
-					mutations.forEach( function( mutation ) {
+						// Reconcile differences for Chinese (Simplfified)'s language code
+						// Note: WET uses "zh-Hans", Twitter uses "zh-ch" and falls back to English if the former is used
+						if ( twitterLinkLang.toLowerCase() === "zh-hans" ) {
+							twitterLinkLang = "zh-cn";
+						}
 
-						// Deal only with removed HTML nodes
-						mutation.removedNodes.forEach( function( removedNode ) {
+						// Set the widget to the link's language
+						twitterLink.dataset.lang = twitterLinkLang;
+					}
+				}
```

Comment:
Turns out that most of this logic is pointless. Twitter's widget script is already capable of "re-using" the closest parent element's language on its own.

The English widget UI in the [French demo page](https://wet-boew.github.io/v4.0-ci/demos/twitter/twitter-fr.html)'s example threw me off :P. Most of that page's content is wrapped within ``<div lang="en">``... which caused Twitter's script to go with English. I mistook it as Twitter arbitrarily defaulting to English unless ``data-lang`` is explicitly-set.

My next force-push will only keep the Chinese (Simplified) replacement logic at this spot.